build/packages-template/bbb-html5/mongod.service

```diff
@@ -1,39 +0,0 @@
-#
-# Override file for mongod.service
-#
-[Unit]
-Description=High-performance, schema-free document-oriented database
-After=network.target
-Documentation=https://docs.mongodb.org/manual
-PartOf=bigbluebutton.target
-
```

Comment:
A few more mentions of mongo are found in https://github.com/bigbluebutton/bigbluebutton/blob/v3.0.x-release/bigbluebutton-config/bin/bbb-conf
Let's remove them.